src/main/java/entities/Target.java

```diff
@@ -0,0 +1,55 @@
+package entities;
+
+import java.sql.Date;
```

Comment:
There is a discrepancy here in that there is a different Date type being used throughout the project. Maybe you can change this to java.util.date instead so that everything can be consistent throughout.

Reply:
Change made